lumen/ai/coordinator.py

```diff
@@ -204,17 +204,81 @@ def on_rerun(instance, _):
         def on_clear(instance, _):
             self._memory.cleanup()
 
+        def on_submit(event=None, instance=None):
+            objects = []
+            chat_input = self.interface.active_widget
+
+            # Process uploaded files through SourceControls if any exist
+            if chat_input.value_uploaded:
+                source_controls = SourceControls(
+                    downloaded_files={key: io.BytesIO(value["value"]) for key, value in chat_input.value_uploaded.items()},
+                    memory=self._memory,
+                    replace_controls=False,
+                    clear_uploads=True  # Clear the uploads after processing
+                )
+                source_controls.param.trigger("trigger_add")
+                objects.extend(chat_input.views)
+                chat_input.value_uploaded = {}
+
+            if chat_input.value:
+                objects.append(Markdown(chat_input.value))
+
+            if not objects:
+                return
+            value = Column(*objects) if len(objects) > 1 else objects[0]
+            instance.send(value=value, respond=chat_input.value)
+
+            if self._main_placeholder != instance:
+                self._main_placeholder.update(instance)
+
+        log_debug("New Session: \033[92mStarted\033[0m", show_sep="above")
+
+        # Create welcome
+        welcome_text = Typography(
+            "# Illuminate your data\nUpload your dataset to begin, then ask any question, or select a quick action below.",
+            css_classes=["welcome-text"],
+            disable_anchors=True,
+        )
+
+        # Create the welcome screen layout
+        welcome_screen = Column(
+            welcome_text,
+            css_classes=["welcome-screen"],
+            sizing_mode="stretch_both",
+            align="center",
+            styles={
+                "padding": "40px 20px",
+                "max-width": "800px",
+                "margin": "0 auto",
+                "border-radius": "12px",
+                "border": "1px solid #e0e0e0",
+                "background-color": "#ffffff"
+            }
```

Comment:
Let's stick with Mui design language where possible, in this case we can use `Paper`.